agent/auto-config/auto_encrypt.go

```diff
@@ -46,7 +46,7 @@ func (ac *AutoConfig) autoEncryptInitialCertsOnce(ctx context.Context, csr, key
 	}
 	var resp structs.SignedResponse
 
-	servers, err := ac.joinHosts()
+	servers, err := ac.autoEncryptHosts()
```

Comment:
I renamed this function to better describe its purpose, and to more closely draw the parallel between it and the autoConfigHosts function (of which it is partially a copy/paste).